inference-engine/src/plugin_api/cpp_interfaces/impl/ie_infer_request_internal.hpp

```diff
@@ -228,13 +228,29 @@ class InferRequestInternal : virtual public IInferRequestInternal {
      * @param serial Whether to use multiple threads to execute the step
      */
     void execDataPreprocessing(InferenceEngine::BlobMap& inputs, bool serial = false) {
```

Comment:
why `inputs`? I suppose they are outputs?

Reply:
renamed to `preprocessedBlobs`